ajax/libs/react-datagrid/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "react-datagrid",
+  "version": "3.0.0",
+  "filename": "index.js",
+  "description": "A carefully crafted DataGrid for React",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zippyui/react-datagrid.git"
+  },
+  "keywords": [
+    "grid",
+    "datagrid",
+    "data-grid",
+    "widget",
+    "table",
+    "list",
+    "react-component",
+    "react"
+  ],
+  "author": {
+    "name": "ZippyUI",
+    "email": "contact@zippyui.com"
+  },
+  "license": "MIT",
+  "npmName": "react-datagrid",
+  "npmFileMap": [
+    {
+      "basePath": "dist",
+      "files": [
+        "!(react.|react-dom.)*.+(js|css|map)"
```

Comment:
Why so widely exclude `react.*` ? Shouldn't you just want to exlucde `react.min.js` & `react-dom.min.js`